install/kube-rbac-proxy-kafka.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="kube-rbac-proxy-kafka"]
+= Configuring kube-rbac-proxy for Kafka
+:context: kube-rbac-proxy-kafka
+
+toc::[]
+
+The `kube-rbac-proxy` component provides internal authentication and authorization capabilities for Knative Kafka.
```

Comment:
```suggestion
The `kube-rbac-proxy` component provides internal authentication and authorization capabilities for Knative for Apache Kafka.
```

Reply:
Same suggestion: "Knative Kafka".

Reply:
(so, the suggestion is to leave as is in this case)

Reply:
Resolved in favor of "Knative for Apache Kafka".